src/resources/commandTemplates/templates.ActionCommand.ts

```diff
@@ -576,7 +617,11 @@ export default class ActionCommand extends ResponsiveSlashCommandSubcommandBuild
       return;
     }
 
-    if (!action[3].sendNoticeFirst) await sendNotice(USER, LOG, interaction);
+    if (!action[3].sendNoticeFirst && !action[3].noNotice) {
+      await sendNotice(USER, LOG, interaction);
+    } else if (action[3].noNotice) {
+      await interaction.followUp(`Successfully added note for ${member}`);
+    }
```

Comment:
this seems to be some action-specific code in a place where different actions could theoretically end up (by disabling notice on something else, for example if we disabled it on mature role additions as it being there is somewhat of an implementation detail)... can you change this to use the past tense of the action taken?